frb_codegen/src/binary/commands_parser.rs

```diff
@@ -83,26 +84,68 @@ mod tests {
         configure_opinionated_test_logging();
         set_cwd_test_fixture("binary/commands_parser/pubspec_yaml")?;
 
-        let config = run_command_line(vec!["", "generate"]);
+        let config = run_command_line::<Config, anyhow::Error>(vec!["", "generate"])?;
         assert_eq!(config.rust_input, "hello.rs".to_string());
         assert!(!config.dart3.unwrap());
 
         Ok(())
     }
 
+    #[test]
+    #[serial]
+    fn test_compute_codegen_config_mode_from_files_auto_pubspec_yaml_faulty() -> anyhow::Result<()>
+    {
+        configure_opinionated_test_logging();
+        set_cwd_test_fixture("binary/commands_parser/faulty_pubspec_yaml")?;
+
+        let result = run_command_line::<Config, anyhow::Error>(vec!["", "generate"]);
+        assert!(result.is_err());
+        assert!(result
+            .unwrap_err()
+            .source()
+            .unwrap()
+            .to_string()
+            .contains("misspelled_dart3"));
+
+        Ok(())
+    }
+
     #[test]
     #[serial]
     fn test_compute_codegen_config_mode_config_file() -> anyhow::Result<()> {
         configure_opinionated_test_logging();
         set_cwd_test_fixture("binary/commands_parser/config_file")?;
 
-        let config = run_command_line(vec!["", "generate", "--config-file", "hello.yaml"]);
+        let config = run_command_line::<Config, anyhow::Error>(vec!["", "generate", "--config-file", "hello.yaml"])?;
         assert_eq!(config.rust_input, "hello.rs".to_string());
         assert!(!config.dart3.unwrap());
 
         Ok(())
     }
 
+    #[test]
+    #[serial]
+    fn test_compute_codegen_config_mode_config_file_faulty_file() -> anyhow::Result<()> {
+        configure_opinionated_test_logging();
+        set_cwd_test_fixture("binary/commands_parser/flutter_rust_bridge_yaml")?;
+        // let result = std::panic::catch_unwind(|| {
```

Comment:
```suggestion
```

Reply:
Oh! Thanks, overlooked that! The comment below is unnecessary as well, since I changed it to propagate Result objects instead panicking. Removed that as well.
Funny, the code coverage dropped by one line - I guess it was that one. I will wait for the CI to finish before closing this thread.

Reply:
That was it - code coverage is good again (nice side effect to spot overlooked commented code lines :))!
There is a failing dart test, but I am sure that is because of #1464.
@fzyzcjy ready to be merged :)

Reply:
Oh, it looks like I was wrong - the codecov/project job still fails :(
Indeed, there is one new code line, but new tests - though I assume codecov doesn't understand that these tests cover it.
How can I configure this? Or do we just ignore it here?